src/backend/joanie/core/models/products.py

```diff
@@ -145,22 +145,18 @@ def get_equivalent_course_run_data(self, visibility=None):
         an equivalent course run with null dates and hidden visibility.
         """
         site = Site.objects.get_current()
-        aggregate = self.target_course_runs.aggregate(
```

Comment:
same here.

Reply:
From my point of view, here there are no impacts as `self.target_course_runs` returns a queryset, the db query is executed only when we access to `aggregate.items`